presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -0,0 +1,25 @@
+====================
+Black Hole Connector
+====================
+
+The Black Hole connector works in similar way as /dev/null device file in Linux operating system. 
```

Comment:
I (or someone else in the Boston office) can go through this and point out wording changes at least.